src/AEG/CutNode.ts

```diff
@@ -122,6 +122,71 @@ export class CutNode {
         }
     }
 
+    /**
+     * Recursive method to return the lowest level node containing the given point.
+     * @param incomingPoint The given point on the canvas.
+     * @returns The lowest node containing the node on the tree.
```

Comment:
You need to document under what circumstances this function returns null, as you claim it does in the return value.

Reply:
This is all my bad, I kind of just focused on adding the reviews without really thinking, will add all of these when I wake up tomorrow.